css/obBase.css

```diff
@@ -195,13 +195,17 @@ form {
 #obContainer {
   transition: width 0.5s ease;
   width: calc(100% - 47px);
-
+  transition: width 0.5s ease;
 }
 
-#obContainer.compressed {
+#ov1 .chatOpen #obContainer {
   width: calc(100% - 231px);
 }
 
+#ov1 .chatOpen #obContainer::-webkit-scrollbar {
+  width: 0 !important;
```

Comment:
I could try. I didn't put that there. I just moved this style and that was already in place.